chroot-bin/crouton-unity-autostart

```diff
@@ -17,9 +17,9 @@ execfromconf() {
     fi
 }
 
-# Upstart launches unity on utopic, and this needs to be started before
+# Upstart launches unity on utopic+, and this needs to be started before
 # any dependent services.
-if [ "$RELEASE" = 'utopic' ]; then
+if [ "$RELEASE" = 'xenial' ]; then
```

Comment:
Yakkety has more strange unity breakage. I'd say let's just get the LTS working first.